tests/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test/assets/TestClasses/UA0013.Fixed.vb

```diff
@@ -1,11 +1,12 @@
 ﻿Imports System.Web.Http
+Imports System.Web.Mvc
 Imports System.Collections.Generic
 Imports Microsoft.AspNetCore.Mvc
 
 Namespace TestProject.TestClasses
 
     Public Class ValuesController2
-        Inherits Controller
+        Inherits ControllerBase
```

Comment:
Same as above, I think I'm missing something

Reply:
Yea, I made that change intentionally because I think it's the more correct translation to Core. Let me know if you disagree; it would be very easy to switch back.